src/HouseofCat.RabbitMQ/Messages/PublishReceipt.cs

```diff
@@ -3,18 +3,17 @@ namespace HouseofCat.RabbitMQ
     public interface IPublishReceipt
     {
         bool IsError { get; set; }
+        public string MessageId { get; set; }
```

Comment:
Oops the public probably shouldn’t be on an interface right?

Reply:
No, its a mistake I will fix.

Publishing bug had me frazzled.